hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/RPCBasicMessageListener.java

```diff
@@ -128,12 +131,10 @@ public void onMessage(Message message) {
             msglog.errorFailedToSendResponse(e);
             return;
         } finally {
-            if (consumerConnectionContext != null) {
-                try {
-                    consumerConnectionContext.close();
```

Comment:
@jmazzitelli If you close consumerConnectionContext then it is impossible to receive next incoming messages. Should be this listener closed after receiving first message?